cnf-certification-test/identifiers/identifiers.go

```diff
@@ -525,6 +525,11 @@ var (
 		Url:     formTestURL(common.LifecycleTestKey, "cpu-isolation"),
 		Version: versionOne,
 	}
+	TestContainerPortNameFormat = claim.Identifier{
```

Comment:
you could also use the AddCatalogEntry to add a new TC

Reply:
This can be done in a different PR by someone else.